### DISCUSSION TITLE
flyway@9.10.2: use Maven metadata

### DISCUSSION THREAD

**Discussion by demcgovern** [ScoopInstaller/Scoop#5317] [General]
Flyway was stuck at 9.8 for a while. Update to 9.10.2 was pulled today, but uses release notes page for checkver. I think it should use the Maven metadata, which has a `release` element under the `versioning` element. I made the change locally yesterday, and it worked. I think it will work more reliably than release notes.

[Maven metadata](https://repo1.maven.org/maven2/org/flywaydb/flyway-commandline/maven-metadata.xml)
[Branch](https://github.com/ScoopInstaller/Main/compare/master...demcgovern:scoop-installer-main:flyway)

```
"checkver": {
    "url": "https://repo1.maven.org/maven2/org/flywaydb/flyway-commandline/maven-metadata.xml",
    "regex": "<release>(\\S+)</release>"
}
```

**Comment by HUMORCE** [comment]
Just commit the PR.

**Reply by demcgovern** [reply]
Merged yesterday.
